README.md

```diff
@@ -14,10 +14,20 @@ The Propel UI Kit is a data visualization and dashboard React component library
 
 ## Install
 
+Note: check the [Migration guide](#migration-guide) in case you installed any of @propeldata/react-prefixed dependencies before.
```

Comment:
Nit: 
```suggestion
Note: check the [Migration guide](#migration-guide) in case you installed any of the "@propeldata/react"-prefixed dependencies before.
```